craft_parts/packages/deb.py

```diff
@@ -35,10 +35,17 @@
 from .deb_package import DebPackage
 from .normalize import normalize
 
-if sys.platform == "linux":
-    # Ensure importing works on non-Linux.
+# Ensure importing works on non-Linux or linux without apt.
+# If the import fails provide a stub that appropriately fails
```

Comment:
What happens "if the import fails" is misleading (probably remained there from a previous implementation)